### DISCUSSION TITLE
Question about an old CC

### DISCUSSION THREAD

**Discussion by BlueSky-fur** [BlackWolfWoof/yagpdb-cc#62] [General]
Didn't you have a Custom command that when someone sends a message link in a channel that then the bot sends an embed that shows the message?

Im either blind or it got removed

**Comment by Wolveric** [comment]
https://github.com/BlackWolfWoof/yagpdb-cc/blob/master/Misc/message_link.yag

**Comment by BlueSky-fur** [comment]
> https://github.com/BlackWolfWoof/yagpdb-cc/blob/master/Misc/message_link.yag

Thank you! So I was blind 🙈

Thanks a lot! :3
